src/dependencies.jl

```diff
@@ -51,7 +49,7 @@ function get_project_deps(project_file::AbstractString; include_jll::Bool=false)
                 dep_entry = convert(String, strip(get(compat, name, "")))
 
                 if !isempty(dep_entry)
-                    compat_entry.version_spec = VersionSpec(dep_entry)
+                    compat_entry.version_spec = semver_spec(dep_entry)
```

Comment:
Changed this because it needs to handle parsing stuff like `1.2, 2.1`, `= 1.3`, `>= 2.4`, etc. `VersionSpec` only deals with single version strings such as `1.2`, `2.3`, etc